### DISCUSSION TITLE
Flow settings

### DISCUSSION THREAD

**Discussion by Shaynan** [phnxdrgn536/FLSun-v400#9] [Q&A] [answered]
Hello !

How can i change my flow settings for all PLA profiles in one shot ?

i want my flow to be 98% for all PLA profiles.

Thanks a lot

Steve

**Comment by phnxdrgn536** [comment, marked as answer]
open each profile file you want to edit permanently with a text editor such as notepad or notepad++ and add the line "material_flow = 98" (with no "" ) under the [values] section and save them. this will permanently change the default value. otherwise you can still do custom settings in cura before you slice the model.

**Comment by Shaynan** [comment]
Thank you very much !

Steve

**Reply by phnxdrgn536** [reply]
No problem, if there's anything else just let me know
